zenguard/zenguard.py

```diff
@@ -73,18 +73,19 @@ def __init__(self, config: ZenGuardConfig):
             raise ValueError(f"LLM {config.llm} is not supported")
 
     def detect(self, detectors: list[Detector], prompt: str):
-        if len(detectors) == 0:
-            return {"error": "No detectors were provided"}
         try:
             response = httpx.post(
-                self._backend + detectors[0].value,
-                json={"messages": [prompt]},
+                self._backend + "v1/detect",
```

Comment:
given this change raw API based detection and client based  APIdetection are diverging in terms of detection implementation.

We should make the detection implementation exactly the same between 2 ways of accessing our API.

Of course, not in this PR